### PR TITLE
Replace mandrill with sendgrid and fix heroku addons

### LINKED ISSUE #493: Move back from Mandrill to Sendgrid
Mandrill on Heroku has no more start plan and the initial plan costs 9.95 dollars.

To keep mandrill setup on pah will wind up on setup process break, we should move fast about it.

### PR DESCRIPTION
**Pull request by thiagogabriel**
Mandrill does not have a free plan anymore, so the process is breaking. Sendgrid has a free plan and became the default configuration.

Some addons changed their names on heroku, so I had to change it on pah.

Closes #493

### CODE REVIEW

**Comment by rodrigopinto** [comment]
@thiagogabriel de uma olhada nos conflitos.

**Comment by thiagogabriel** [comment]
@rodrigopinto acredito que seja no changelog. Vou resolver

**Comment by thiagogabriel** [comment]
@rodrigopinto :shipit:

File: lib/pah/templates/heroku.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class HerokuApp < Rails::Generators::AppGenerator
-  DEFAULT_ADDONS = %w(heroku-postgresql:dev logentries
-                      mandrill:starter rollbar newrelic:wayne librato)
+  DEFAULT_ADDONS = %w(heroku-postgresql logentries sendgrid rollbar newrelic
```
Comment (maurogeorge) [inline lib/pah/templates/heroku.rb]: It is `newrelic` only?

File: lib/pah/templates/sendgrid.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    password:             ENV['SENDGRID_PASSWORD'],
+    domain:               'heroku.com',
+    enable_starttls_auto: true
+  }
+SENDGRID
+
+        inject_into_file 'config/environments/production.rb', sendgrid_config,  before: /^end/, verbose: false
+
+        git add: 'config/environments/production.rb'
+        git_commit 'Add sendgrid config.'
```
Comment (maurogeorge) [inline lib/pah/templates/sendgrid.rb]: `Add Sendgrid config.`  what you think?

File: lib/pah/templates/heroku.rb (hunk 1 above)
Comment (thiagogabriel) [inline lib/pah/templates/heroku.rb]: Yes. I checked yesterday and there is no other free option.

File: lib/pah/templates/sendgrid.rb (hunk 2 above)
Comment (thiagogabriel) [inline lib/pah/templates/sendgrid.rb]: On it! :+1: @maurogeorge

File: lib/pah/templates/heroku.rb (hunk 1 above)
Comment (rodrigopinto) [inline lib/pah/templates/heroku.rb]: Yep, i've checked today doing the code review.

**Pull request merged**